src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -48,7 +49,7 @@
 
   <PropertyGroup Label="NuGet and AssemblyInfo metadata">
     <!--This should be passed from the VSTS build-->
-    <MsalClientSemVer Condition="'$(MsalClientSemVer)' == ''">4.46.0</MsalClientSemVer>
+    <MsalClientSemVer Condition="'$(MsalClientSemVer)' == ''">$(MsalInternalVersion)</MsalClientSemVer>
```

Comment:
We need to add a check to see we are not setting a version higher than the NuGet released version

Reply:
This is just a default version string used if the CI isn't injecting the version. Not sure it matters too much.

Reply:
sounds good